core/domain/collection_domain_test.py

```diff
@@ -1077,3 +1079,38 @@ def test_validation_fails_with_invalid_contributor_id_in_contributor_ids(
             utils.ValidationError,
             'Expected each id in contributor_ids to be string, received 2'):
             self.collection_summary.validate()
+
+    def test_is_private(self):
+        self.assertTrue(self.collection_summary.is_private())
+        self.collection_summary.status = constants.ACTIVITY_STATUS_PUBLIC
+        self.assertFalse(self.collection_summary.is_private())
+
+    def test_is_solely_owned_by_user_one_owner(self):
+        self.assertTrue(
+            self.collection_summary.is_solely_owned_by_user('owner_id'))
+        self.assertFalse(
+            self.collection_summary.is_solely_owned_by_user('other_id'))
+        self.collection_summary.owner_ids = ['other_id']
```

Comment:
Same thing as above, let's have a setter for these to preserve abstraction barriers.